spec/spec_helper.rb

```diff
@@ -37,6 +37,9 @@
   config.include ModelReconstruction
   config.include TestData
   config.extend VersionHelper
+  config.filter_run focus: true
+  config.filter_run_excluding blur: true
+  config.run_all_when_everything_filtered = true
```

Comment:
We don't need this change for this bugfix; I just like to run a single test without screwing around on the command line looking for line numbers and typing out file names. I've removed the commit.